peg-highlight/pmh_parser.c

```diff
@@ -563,10 +563,11 @@ static pmh_element *ll_mergesort(pmh_element *list,
                                  int (*compare)(const pmh_element*,
                                                 const pmh_element*))
 {
+	pmh_element *out_head;
     if (!list)
         return NULL;
     
-    pmh_element *out_head = list;
+	out_head = list;
```

Comment:
I'm sorry for that, coding style is very import, I will be attention of this next time.